src/main/webapp/WEB-INF/templates/search.jsp

```diff
@@ -17,17 +19,28 @@
       </div>
       <div class="card-container">
         <c:forEach items='${requestScope["cardList"]}' var="knowledgeCard">
-          <div class="card">
-            <header>
-              <img class="icon" src="${knowledgeCard.getIcon()}" alt="${knowledgeCard.getSource()} site icon" width="25px" height="25px">
-              <a href="${knowledgeCard.getLink()}" class="card-title">
-                ${knowledgeCard.getTitle()}
-              </a>
-            </header>
-            <c:if test="${knowledgeCard.getCode() != null}">
-              <pre><code>${knowledgeCard.getCode()}</code></pre>
-            </c:if>
-            <p>${knowledgeCard.getDescription()}</p>
+          <div class="card" url="${knowledgeCard.getLink()}">
+            <div class="vote">
+              <button class="upvote">
+                <i class="fas fa-chevron-up"></i>
+              </button>
+              <span class="total-votes">${knowledgeCard.getVotes()}</span>
+              <button class="downvote">
+                <i class="fas fa-chevron-down"></i>
+              </button>
+            </div>
+            <div class="content">
```

Comment:
During this refactoring, the padding for the card got moved outside of the card content, so now the scrollbar appears over the card content. Can you move the padding back inside?